### DISCUSSION TITLE
Using code-server in browser, and login to gituhb. Where code-server store the github credential ?

### DISCUSSION THREAD

**Discussion by WeBeautifulNow** [coder/code-server#6566] [Q&A] [answered]
I build a jupyter notebook service and install code-server.
So I can use vscode copilot for jupyter notebook.
After I login to github, I can use copilot. I want to know where the github login credential stored in code-server. 
I find many place like ~\.config\code-server, but I can't find the credential.

**Comment by code-asher** [comment, marked as answer]
VS Code uses browser storage for credentials.  For example, if you use Chrome open the dev tools, navigate to `Application`, expand `Local Storage`, then click on the URL underneath.  You should see a `secrets.provider` key.  The value is encrypted.
